modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm

```diff
@@ -146,16 +136,16 @@
 	alt_titles = list("Security Officer", "Security Operative", "Peacekeeper")
 
 /datum/job/security_sergeant
-	alt_titles = list("Security Sergeant", "Security Squad Leader", "Security Task Force Leader", "Security Fireteam Leader")
+	alt_titles = list("Security Sergeant", "Security Squad Leader", "Security Task Force Leader", "Security Fireteam Leader", "Security Enforcer") //why are these all so edgy, you're a mall cop
 
 /datum/job/shaft_miner
-	alt_titles = list("Shaft Miner", "Excavator")
+	alt_titles = list("Shaft Miner", "Excavator", "Spelunker", "Drill Technician", "Prospector")
 
 /datum/job/station_engineer
-	alt_titles = list("Station Engineer", "Emergency Damage Control Technician", "Electrician", "Engine Technician", "EVA Technician")
+	alt_titles = list("Station Engineer", "Emergency Damage Control Technician", "Electrician", "Engine Technician", "EVA Technician", "Engineer", "Mechanic")
 
 /datum/job/virologist
 	alt_titles = list("Virologist", "Pathologist")
 
 /datum/job/warden
-	alt_titles = list("Warden", "Brig Sergeant", "Dispatch Officer")
+	alt_titles = list("Warden", "Brig Sergeant", "Dispatch Officer", "Brig Governor", "Jailer", "Gaoler")
```

Comment:
I legit had to search up what a "Gaoler" is. I don't think that it's good, because it makes the role more cryptic in what it's supposed to be. People shouldn't have to check on Google to figure out which job that person has.

Reply:
that was a tom request actually

Reply:
yeah i changed it to brig gaoler so people have a clue anyway